File: src/main/java/schema/TypeMap.java
package schema;

import org.janusgraph.core.attribute.Geoshape;

import java.util.Date;
import java.util.HashMap;
import java.util.UUID;

public class TypeMap {
    public static HashMap<String, Class> MAP = new HashMap<String, Class>();

    static {
        MAP.put("String", String.class);
        MAP.put("Character", Character.class);
        MAP.put("Boolean", Boolean.class);
        MAP.put("Byte", Byte.class);
        MAP.put("Short", Short.class);
        MAP.put("Integer", Integer.class);
        MAP.put("Long", Long.class);
        MAP.put("Float", Float.class);
        MAP.put("Geoshape", Geoshape.class);
        MAP.put("UUID", UUID.class);
        MAP.put("Date", Date.class);
    };
}

File: access-control/src/main/kotlin/schema/TypeMap.kt
package schema

import org.janusgraph.core.attribute.Geoshape
import java.util.*
import kotlin.collections.HashMap

class TypeMap {
    companion object {
        val MAP: HashMap<String, Class<*>> = hashMapOf(
                "String" to String::class.javaObjectType,
                "Character" to Character::class.javaObjectType,
                "Boolean" to Boolean::class.javaObjectType,
                "Byte" to Byte::class.javaObjectType,
                "Short" to Short::class.javaObjectType,
                "Integer" to Int::class.javaObjectType,
                "Long" to Long::class.javaObjectType,
                "Float" to Float::class.java,
                "Geoshape" to Geoshape::class.javaObjectType,
                "UUID" to UUID::class.javaObjectType,
                "Date" to Date::class.javaObjectType
        )
    }
}